pkgs/by-name/tl/tlm/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  pkgs,
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  ollamaVariant ? pkgs.ollama,
+}:
+
+buildGoModule {
+  name = "tlm";
```

Comment:
Missing version

You can use a rev of a slightly older version, make it build then run that script in `maintainers/scripts/update.nix` to update this package

Reply:
Is this something that I would do myself, or something that happens somewhere in CI?

Reply:
You do yourself. When the update script is OK after merge r-ryantm can do most of the work for you.